vnsiclient.c

```diff
@@ -268,16 +259,13 @@ bool cVNSIClient::EpgChange()
       continue;
 
     uint32_t channelId = CreateStringHash(schedule->ChannelID().ToString());
-    it = m_epgUpdate.find(channelId);
-    if (it != m_epgUpdate.end() && it->second.lastEvent >= lastEvent->StartTime())
+    auto it = m_epgUpdate.find(channelId);
+    if (it == m_epgUpdate.end() || it->second.attempts ||
```

Comment:
On 2016/07/05 19:17, Rainer Hochecker <notifications@github.com> wrote:
> > @@ -268,16 +259,13 @@ bool cVNSIClient::EpgChange()
> >        continue;
> >  
> >      uint32_t channelId = CreateStringHash(schedule->ChannelID().ToString());
> > -    it = m_epgUpdate.find(channelId);
> > -    if (it != m_epgUpdate.end() && it->second.lastEvent >= lastEvent->StartTime())
> > +    auto it = m_epgUpdate.find(channelId);
> > +    if (it == m_epgUpdate.end() || it->second.attempts ||
> 
> that changes behaviour. it was used to skip if attempts > 3

Oops.  Stupid, that ">3" got lost.